imports/plugins/core/versions/server/migrations/41_payment_methods.js

```diff
@@ -0,0 +1,43 @@
+import { Migrations } from "meteor/percolate:migrations";
+import { Packages, Shops } from "/lib/collections";
+
+Migrations.add({
+  version: 41,
+
+  up() {
+    const shops = Shops.find();
```

Comment:
Strange... I was testing this as I made it, and the `for-of` definitely works as intended. I wonder if it implicitly calls `Cursor#toArray`. Either way, `Cursor#forEach` would be better because it can operate as results are retrieved.